src/components/common/StaffPayment.vue

```diff
@@ -0,0 +1,152 @@
+<template>
```

Comment:
Quick Question about this component. Is this a new feature for staff on all maintenance filings or is this specific to just Corrections?

Reply:
Sorry, i got confused. This was a relocation of a component from AR to common. Not something i had worked with prior, i see now this is for all filing types. Resolved!

Reply:
The enhancements to Staff Payment apply to all filings... I am working on that in Part 2!